app/controllers/admin/needs_attention_auctions_controller.rb

```diff
@@ -0,0 +1,5 @@
+class Admin::NeedsAttentionAuctionsController < Admin::BaseController
```

Comment:
thoughts on `Admin::Auctions::NeedsAttentionController`to be more consistent with https://github.com/18F/micropurchase/blob/develop/app/controllers/admin/auctions/closed_controller.rb ?